nextflow_schema.json

```diff
@@ -76,36 +76,37 @@
             "properties": {
                 "dehosting_idx": {
                     "type": "string",
-                    "default": "./databases/PhiPacHum_m2.idx",
                     "description": "Mash sketch used for contamination detection and speciation (Sketch comments must be a taxonomic string similar to what Kraken2 outputs)",
                     "pattern": "^\\S+$",
+                    "exists": true,
                     "format": "file-path"
                 },
                 "mash_sketch": {
                     "type": "string",
-                    "default": "./databases/GTDBSketch_20231003.msh",
                     "description": "Minimpa2 index for dehosting and kitome removal",
                     "pattern": "^\\S+$",
+                    "exists": true,
                     "format": "file-path"
                 },
                 "bakta_db": {
                     "type": "string",
                     "description": "Database use for bakta, this value is optional as bakta can be skipped",
-                    "default": "./databases/db-light",
                     "pattern": "^\\S+$",
+                    "exists": true,
                     "format": "directory-path"
                 },
                 "kraken2_db": {
                     "type": "string",
-                    "default": "./databases/k2_standard_20220607/",
                     "description": "Kraken2 database",
                     "pattern": "^\\S+$",
+                    "exists": true,
                     "format": "directory-path"
                 },
                 "staramr_db": {
                     "type": "string",
                     "description": "It is recommended to use the StarAMR database in the StarAMR container however, an external option can be specified",
                     "pattern": "^\\S+$",
+                    "exists": true,
```

Comment:
Shoul we the existence checking here? This parameter is hidden currently anyway

Reply:
We should since it doesn't hurt, and checking for existence is what will trigger a fast error if an invalid path was passed in. Note that if the value is kept as `null`, then existence checking doesnt happen:

> If the parameter is set to null, false or an empty string, this validation is ignored. It does not check if the path exists.

From https://nextflow-io.github.io/nf-validation/nextflow_schema/nextflow_schema_specification/#exists

Reply:
I see, I am understanding some of the logic better now. Thank you for taking care of this.